frontend/src/views/HomeView.vue

```diff
@@ -1 +1,66 @@
-<template><h1>Test</h1></template>
+<template>
+  <div class="page-wrapper">
+    <div class="table-container">
+      <h2>Roles</h2>
+      <el-table :data="rolesStore.getRoles" style="width: 100%">
```

Comment:
```suggestion
      <ElTable :data="rolesStore.getRoles" style="width: 100%">
```

Please, inside all components, instead of use snake-case, use CapitalCase